treadmill-aws/Makefile

```diff
@@ -11,6 +11,11 @@ WHEELS=$(BLD)/wheels
 TREADMILL_DIR=$(PWD)/../../treadmill
 TREADMILL_AWS_DIR=$(PWD)/../../treadmill-aws
 
+TREADMILL_AWS_TAG=$(shell git --git-dir=${TREADMILL_AWS_DIR}/.git describe --tags HEAD --long)
```

Comment:
nit - you will need to reuse this code in other projects that build rpm, and as it will be cut/paste, will be easier to keep it "generic" - instead of TREADMILL_AWS_VER - just RPM_VER, RPM_REL, or just VERSION, RELEASE. 

Otherwise looks good.

Reply:
Makes sense, variable names changed to REPO_TAG, REPO_VER, etc.